### PR TITLE
SI-8786 disable part of thest that's failing the jdk8 build

### PR DESCRIPTION
**Pull request by lrytz**
As noted in SI-8786, this test currently fails the [jdk 8 build](https://scala-webapps.epfl.ch/jenkins/view/2.11.x/job/scala-nightly-auxjvm-2.11.x/jdk=jdk8,label=auxjvm/122/console).

### CODE REVIEW

**Comment by lrytz** [comment]
review by @retronym

File: test/files/jvm/varargs/JavaClass.java [hunk 1]
```diff
@@ -11,6 +11,6 @@ public static void callSomeAnnotations() {
 	va.vi(1, 2, 3, 4);
         varargz(5, 1.0, 2.0, 3.0);
 	va.vt(16, "", "", "");
-	System.out.println(va.vt1(16, "a", "b", "c"));
+	// System.out.println(va.vt1(16, "a", "b", "c"));
```
Comment (retronym) [inline test/files/jvm/varargs/JavaClass.java]: Could do with a comment to link to the ticket

**Comment by retronym** [comment]
This reminded me of another problem with generic varargs: https://issues.scala-lang.org/browse/SI-1459?focusedCommentId=55693&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-55693

Not sure it there is any relationship between them, though.

**Comment by retronym** [comment]
s/thest/test

**Comment by retronym** [comment]
Indeed, bounding the generic type argument makes things work:

```
git diff
diff --git a/test/files/jvm/varargs/VaClass.scala b/test/files/jvm/varargs/VaClass.scala
index e94e8a6..83ce163 100644
--- a/test/files/jvm/varargs/VaClass.scala
+++ b/test/files/jvm/varargs/VaClass.scala
@@ -9,5 +9,5 @@ class VaClass {
   @varargs def vs(a: Int, b: String*) = println(a + b.length)
   @varargs def vi(a: Int, b: Int*) = println(a + b.sum)
   @varargs def vt[T](a: Int, b: T*) = println(a + b.length)
-  @varargs def vt1[T](a: Int, b: T*): T = b.head
+  @varargs def vt1[T <: String](a: Int, b: T*): T = b.head
 }
```

**Comment by lrytz** [comment]
PTAL

**Comment by xeno-by** [comment]
@lrytz Both google and urban dictionary are failing me. What does this mean? :)

**Comment by lrytz** [comment]
please take another look :-) http://www.urbandictionary.com/define.php?term=PTAL

**Comment by retronym** [comment]
LGTM

**Pull request merged**